### ISSUE TITLE
Potentially swap to SWA's learning rate schedule for CIFAR baselines 

### ISSUE THREAD

**Issue by dustinvtran** [google/uncertainty-baselines#64]
> @dustinvtran 
> From preliminary experiments, the LR schedule from the SWA papers (https://github.com/timgaripov/swa/blob/master/train.py#L94) seems to improve the baseline results (at least for deterministic and dropout). Upgrading to that one may close the gap from our deterministic baseline which reproduces the original paper of 96.0% (and we get 0.154 NLL). Their papers' baseline reports 96.4% and 0.12 NLL. (Same for CIFAR-100.)

https://github.com/google/edward2/issues/233
